backend/tests/Feature/StyleCriteriaTest.php

```diff
@@ -1,11 +1,11 @@
 <?php
-declare(strict_types=1);
+    declare(strict_types=1);
 
-namespace Tests\Feature;
+    namespace Tests\Feature;
```

Comment:
```suggestion
namespace Tests\Feature;
```
GitHub is forcing me to split up this unindenting suggestion because "suggestions on deleted lines is not allowed". 😕